WordPress/wordpress-playground
# wp-now : Add/support for wpenv config

Related to: https://github.com/WordPress/wordpress-playground/issues/230

This pr supports reading `.wp-env.json`, and `.wp-env.override.json` and converting them to `wp-env.json` format.

## Testing Instructions
- Pull the changes from this branch to your local environment
- Run nvm use
- Run yarn install
- Run yarn build
- Install nx globally `npm install -g nx@latest`
- Ensure that you have a folder with a plugin
- Run `npx nx preview wp-now start --path=<path-to-plugin>`
- Ensure that you see the default options printed out.
- Add a `.wp-env.json` file to your plugin directory:
```
{
	"core": "6.2"
}
```
- Run `npx nx preview wp-now start --path=<path-to-plugin>`
- Make sure that you see `wp: 6.2` in the console
- Create a `.wp-env.override.json`file in your plugin directory:
```
{
	"core": "6.33"
}
```
- Ensure that you see `wp: 6.33` in the console (You will receive an error but that's ok)
